.gitignore

```diff
@@ -81,5 +81,9 @@ npm-debug.*
 *.orig.*
 web-build/
 
+*.tar.gz
```

Comment:
This is fine to add, however we can clean things up via setting the artifacts dir for EAS builds. I'll push that into `package.json` since we already have the `build` directory ignored as well (and we won't have to look for them scattered in the root)

Reply:
fix pushed. resolving